src/Compilers/Core/Portable/PEWriter/BlobWriter.cs

```diff
@@ -321,14 +272,14 @@ internal void WriteDecimal(decimal value)
             value.GetBits(out isNegative, out scale, out low, out mid, out high);
 
             WriteByte((byte)(scale | (isNegative ? 0x80 : 0x00)));
-            WriteUint(low);
-            WriteUint(mid);
-            WriteUint(high);
+            WriteUInt32(low);
+            WriteUInt32(mid);
+            WriteUInt32(high);
         }
 
-        internal void WriteDateTime(DateTime value)
+        public void WriteDateTime(DateTime value)
         {
-            WriteLong(value.Ticks);
+            WriteInt64(value.Ticks);
```

Comment:
it's used in Portable PDB to write date time constants. I'm using the same format as native PDB, which is doing this:

http://source.roslyn.io/#Microsoft.CodeAnalysis/NativePdbWriter/PdbWriter.cs,1191
http://source.roslyn.io/#Microsoft.CodeAnalysis/NativePdbWriter/ISymUnmanagedWriter.cs,a9d0659247f3c0f8